src/covid19sim/configs/naive_config.yml

```diff
@@ -0,0 +1,37 @@
+---
+# EXPERIMENTAL PARAMETERS
+COLLECT_LOGS : True
+COLLECT_TRAINING_DATA : True
+USE_INFERENCE_SERVER : True
```

Comment:
Do we need `COLLECT_LOGS` when `COLLECT_TRAINING_DATA` is `True`. Are you using `logs` now that `clustering` is in the loop?

Reply:
no, I don't think we read in logs anywhere anymore. I think COLLECT_LOGS is about the simulator logs and COLLECT_TRAINING_DATA is about the server outputting logs. I think these should be combined into 1 thing, but I didn't think it was a great idea for me to go refactoring the actual configs too much in this PR. I thought just splitting them out into a YML with multiple options was a sufficient change, without actually changing the behaviour of the parameters.

Reply:
Agree!